fsr.ino

```diff
@@ -473,28 +473,39 @@ class SerialProcessor {
         case 'T':
           PrintThresholds();
           break;
-        default:
+        case '0':
```

Comment:
Can  you try and see if the following works?
```cpp
case '0'...'9': 
```
I recognize that it's non-standard and compiler dependent  but I think since arduino ide is essentially a requirement for this project, if it works then I think it should be fine and we don't need to worry too much about portability? I don't actually know which compiler arduino ide uses.

If it does work can you also add a comment, something like:
```cpp
case '0'...'9':  // Ellipses are non-standard but work in gcc
```